themes/src/main/resources-community/theme/base/admin/messages/messages_de.properties

```diff
@@ -0,0 +1,28 @@
+invalidPasswordMinLengthMessage=Ung\u00FCltiges Passwort: minimal L\u00E4nge {0}.
+invalidPasswordMinLowerCaseCharsMessage=Ung\u00FCltiges Passwort: muss mindestens {0} Kleinbuchstaben beinhalten.
+invalidPasswordMinDigitsMessage=Ung\u00FCltiges Passwort: miss mindestens {0} Zahlen beinhalten.
+invalidPasswordMinUpperCaseCharsMessage=Ung\u00FCltiges Passwort: muss mindestens {0} Grossbuchstaben beinhalten.
+invalidPasswordMinSpecialCharsMessage=Ung\u00FCltiges Passwort: muss mindestens {0} spezielle Zeichen beinhalten.
+invalidPasswordNotUsernameMessage=Ung\u00FCltiges Passwort: darf nicht identisch mit dem Benutzernamen sein.
+invalidPasswordRegexPatternMessage=Ung\u00FCltiges Passwort: stimmt nicht mit Regex-Muster \u00FCberein.
+invalidPasswordHistoryMessage=Ung\u00FCltiges Passwort: darf nicht identisch mit einem der letzen {0} Passw\u00F6rter sein.
+invalidPasswordBlacklistedMessage=Ung\u00FCltiges Passwort: Password ist zu bekannt und auf der schwarzen Liste.
+invalidPasswordGenericMessage=Ung\u00FCltiges Passwort: Neues Passwort erf\u00FCllt die Passwort-Anforderungen nicht.
```

Comment:
All messages above start with a lower-case letter after the :, so this should probably be "neues" instead of "Neues". Or all the other messages need fixing.